docs/recipes/puppeteer.md

```diff
@@ -0,0 +1,59 @@
+# Testing web apps using Puppeteer
+
+## Dependencies
+
+- [Puppeteer](https://github.com/GoogleChrome/puppeteer)
+	- `npm i --save-dev puppeteer`
+
+## Setup
+
+The first step is setting up a helper to configure the environment:
+
+`./test/helpers/setup.js`
+
+```js
+const puppeteer = require('puppeteer');
+const url = "https://google.com"; // App URL, for example, google.com
```

Comment:
It might be neat to be able to pass the URL to the `setup` function instead.

Reply:
Yes, it will be awesome to add some more examples